Assets/Scripts/UI/Overlay/OverlayMap.cs

```diff
@@ -287,10 +270,11 @@ public void SetOverlay(string name)
                 }
 
                 Tile tile = WorldController.Instance.GetTileAtWorldCoord(new Vector3(x, y, z));
-                return (int)script.Call(handle, new object[] { tile, World.Current }).ToScalar().CastToNumber();
+                DynValue result = FunctionsManager.Overlay.Call(descr.LuaFunctionName, new object[] { tile, World.Current });
+                return (int)result.ToScalar().CastToNumber();
```

Comment:
Basically Lua debugging is in a sad state and needs improvement across the board. Might as well make it as good as possible in the files this PR touches.